cmd/transcriber/call/tracks.go

```diff
@@ -0,0 +1,341 @@
+package call
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/mattermost/calls-transcriber/cmd/transcriber/apis/whisper.cpp"
+	"github.com/mattermost/calls-transcriber/cmd/transcriber/config"
+	"github.com/mattermost/calls-transcriber/cmd/transcriber/ogg"
+	"github.com/mattermost/calls-transcriber/cmd/transcriber/opus"
+	"github.com/mattermost/calls-transcriber/cmd/transcriber/transcribe"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/rtcd/client"
+
+	"github.com/pion/webrtc/v3"
+)
+
+const (
+	trackInAudioRate          = 48000                                            // Default sample rate for Opus
+	trackAudioChannels        = 1                                                // Only mono supported for now
+	trackOutAudioRate         = 16000                                            // 16KHz is what Whisper requires
+	trackInAudioSamplesPerMs  = trackInAudioRate / 1000                          // Number of audio samples per ms
+	trackOutAudioSamplesPerMs = trackOutAudioRate / 1000                         // Number of audio samples per ms
+	trackAudioFrameSizeMs     = 20                                               // 20ms is the default Opus frame size for WebRTC
+	trackInFrameSize          = trackAudioFrameSizeMs * trackInAudioRate / 1000  // The input frame size in samples
```

Comment:
minor, but this made more sense to me in my head:
```suggestion
	trackInFrameSize          = trackAudioFrameSizeMs * trackInAudioSamplesPerMs  // The input frame size in samples
```

Reply:
Yeah, I added some of those later on so it makes sense to simplify. Thanks!